README.md

```diff
@@ -128,6 +128,8 @@ accelerate launch train_dreambooth.py \
 Try out the 🤗 Gradio Space which should run seamlessly on a T4 instance:
 [smangrul/peft-lora-sd-dreambooth](https://huggingface.co/spaces/smangrul/peft-lora-sd-dreambooth).
 
+>Tips: If you find this space is paused, please clone it and run it in your own space. Note that some applicable Python environments are: peft==0.0.1, diffusers==0.12.0, and transformers==4.26.0.
```

Comment:
I don't think we need to put this as a quote, just a follow up sentence to the previous one should be enough. Regarding the last sentence, I don't quite understand it, do you mean this space only runs with these exact versions?

Reply:
> Sorry for bothering you

Oh no, not a bother at all.

> The original author of this demo did not specify the dependent versions, which leads to errors when duplicating the environment.

Hmm, that's not good. I really wouldn't want users to have to install peft 0.0.1 for the space to work. This makes me think we should either remove the reference to the demo, as it's basically broken, or update and re-activate the demo, so that it works with the latest versions of the mentioned libraries.

@pacman100 do you have an opinion on that?

Reply:
Thank you, it seems this is an issue that needs to be discussed among the maintainers, but if my help is needed, I will do my best.